README.rst

```diff
@@ -26,22 +46,61 @@ advanced settings.
    ``"invideoquiz"`` to the policy value list.
 3. Click the "Save changes" button.
 
-Package Requirements
+Configuration in Studio
+-----------------------
+
+``Video Location``: The ID of the Video Component, eg:
+    ``cf5557d7eee9408786116f9fead8b2ba``
+``Problem Timemap``: A JSON dictionary with:
+    - timestamps (in seconds, as a string) as keys
+    - a problem component ID as the value, eg:
+    ``{"5": "10fd296e593e404ba999d78c6a76db84"}``
+
+This would display the Problem ``10fd29...`` 5 seconds into Video ``cf5557...``.
+
+TODO: This should be further fleshed out to explain how to get these
+values, etc.
+
+Testing in LMS
+--------------
+
+To see the full behavior, you must use the masquerade feature to
+impersonate a normal learner.
+
+When viewing the Unit as a Staff/Instructor, you will instead see a
+debug note that displays at which point the video will be paused for the
+problem.
+
+Implementation Notes
 --------------------
 
-setup.py contains a list of package dependencies which are required for this XBlock package.
-This list is what is used to resolve dependencies when an upstream project is consuming
-this XBlock package. requirements.txt is used to install the same dependencies when running
-the tests for this package.
+This approach is both clever and brittle.
 
-License
--------
+With a small amount of code, we're able to leverage the functionality of
+existing problems and the video player into a fairly elegant solution.
 
-The In Video Quiz XBlock is available under the AGPL Version 3.0 License.
+But as it depends upon expected behavior and display in the LMS,
+it requires a specific configuration
+and the behavior is vulnerable to breaking as the LMS experience changes
+(eg, the conversion to the Learning MFE from the Legacy Courseware).
```

Comment:
Does this block work in the learning MFE?

Reply:
Would this block work with any video player or just the current default Open edX video player? (I ask as I'm proposing a blended project to overhaul or replace the current player)

Reply:
It works in the MFE _now_; it didn't until I fixed it/added support last week.

Reply:
In theory, it could be made to work.
I think you just need a basic play/pause API to interact with the video player.